native/src/components/Header.tsx

```diff
@@ -196,6 +199,7 @@ const Header = ({
           ? [renderOverflowItem(HeaderButtonTitle.Location, () => navigation.navigate(LANDING_ROUTE))]
           : []),
         renderOverflowItem(HeaderButtonTitle.Settings, () => navigation.navigate(SETTINGS_ROUTE)),
+        ...[renderOverflowItem(t(HeaderButtonTitle.ContrastTheme), toggleTheme)],
```

Comment:
❓ Why did you choose to move this to the header overflow menu? I think this should go to the app settings.

Reply:
It was the previous version, so I have not yet implemented it